scratchapi2/user.py

```diff
@@ -1,47 +1,53 @@
-#pylint: disable=trailing-whitespace
 """
 Scratch API User
 
 Anything returned from API does not contain any attributes.
 Use info() to set/get it.
 
+Contains classes:
+* Project
+* User
+* Classroom
 """
-#pylint: enable=trailing-whitespace
 
 import requests
 from .excs import ScratchAPIError
 from .gclass import GenericData
 
+def _request(path, *opts, api_url="https://api.scratch.mit.edu/"):

```

Comment:
This is used in more than one class, so I defined it as a module-level function.